### DISCUSSION TITLE
Version 6.7.6?

### DISCUSSION THREAD

**Discussion by thetumper** [openapi-ts/openapi-typescript#1677] [General]
npm outdated shows there is a 6.7.6 for openapi-typescript.  But, the tags here do not include it?  Is it fully released?  Missed tagging?  Thanks.

**Comment by drwpow** [comment]
Oh whoops! Automated script missed tagging and GitHub releases. Currently in the process of preparing for 7.x RC so some things aren’t working as expected (but will be back to normal soon).

Just published release here: https://github.com/drwpow/openapi-typescript/releases/tag/openapi-typescript%406.7.6

**Reply by thetumper** [reply]
Thanks!
